HDF5Examples/C/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required (VERSION 3.12)
-project (HDF5Examples_C)
+PROJECT (HDF5Examples_C C)
```

Comment:
keep lowercase project

Reply:
There are 4 files with PROJECT that should be lowered case and H5PAR file should also add a "C" language to its project command.